finalPrep/README.md

```diff
@@ -23,51 +23,51 @@ Some things to consider as you refactor include...
 
 Take your time as you go back and refactor your exercises from each section. We've included a handy checklist for you to go through below.
 
-- [ ] I have refactored my `section1` exercises to the best of my ability
-- [ ] I have refactored my `section2` exercises to the best of my ability
-- [ ] I have refactored my `section3` exercises to the best of my ability
+- [x] I have refactored my `section1` exercises to the best of my ability
+- [x] I have refactored my `section2` exercises to the best of my ability
+- [x] I have refactored my `section3` exercises to the best of my ability
 
 ## Gearing Up for Gear Ups
 
 Gear Up is a part of our curriculum where we talk about topics such as privilege and power, and ethical issues in the tech industry. 
 
-- [ ] [Complete this assignment](https://github.com/turingschool/gear-up/blob/main/m0/Intro.To.GearUp.md)
-  - [ ] Add a link for your gist here: 
+- [x] [Complete this assignment](https://github.com/turingschool/gear-up/blob/main/m0/Intro.To.GearUp.md)
+  - [x] Add a link for your gist here: [Unconscious_Bias_Gear_Up](https://gist.github.com/DanielN88/47439cc44eecd5e1a9aeaefc2c463e0b) 
 
 ## Time Management Prep
 
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
 
-- [ ] Standard Mod 1 class schedule (find the Mod 1 calendar [on the Front End curriculum site](https://frontend.turing.edu/))
-- [ ] Study/Project work time (usually about 25 hours/week outside of class time)
-- [ ] Personal, Health + Wellness
+- [x] Standard Mod 1 class schedule (find the Mod 1 calendar [on the Front End curriculum site](https://frontend.turing.edu/))
+- [x] Study/Project work time (usually about 25 hours/week outside of class time)
+- [x] Personal, Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- <img width="1440" alt="Screen Shot 2022-01-15 at 12 53 13 PM" src="https://user-images.githubusercontent.com/92230099/149637708-b5793378-b36a-480f-acbf-42cf3cf29128.png">
+- <img width="1440" alt="Screen Shot 2022-01-15 at 12 53 20 PM" src="https://user-images.githubusercontent.com/92230099/149637732-7e1243dd-00ef-4601-8932-47235d8d70bf.png">
+- <img width="1440" alt="Screen Shot 2022-01-15 at 12 53 23 PM" src="https://user-images.githubusercontent.com/92230099/149637741-c6937a6f-9727-49c3-9357-9d5fb59c08c7.png">
```

Comment:
Your calendar is looking good! Nice work carving out time for yourself while keeping plenty of time for work. I look at my calendar as a starting point that can flex throughout the day as I realize I need more/less time than what I budgeted. So definitely look for ways to change and improve your time management as you go!